apollo-portal/src/main/resources/static/views/component/namespace-panel-master-tab.html

```diff
@@ -32,8 +32,8 @@
             <div class="col-md-6 col-sm-6 text-right header-buttons">
 
                 <button type="button" class="btn btn-default btn-sm" data-tooltip="tooltip" data-placement="bottom"
-                    title="{{'Component.Namespace.Master.LoadNamespaceTips' | translate }}"
-                    ng-click="refreshNamespace()">
+                        title="{{'Component.Namespace.Master.LoadNamespaceTips' | translate }}"
+                        ng-click="refreshNamespace()">
```

Comment:
I do not know why there are some diffs, in my PC local, the line is not changed.